tests/test_es_models.py

```diff
@@ -23,7 +26,7 @@ def test_missing_required_args(input_args):
 def test_get_es_models_standard_behavior():
     models = get_es_models()
 
-    assert len(models) == 11
+    assert len(models) == 12
```

Comment:
it's a little weird that we have mutation indices unfolded but graph indices folded under 'gdc_from_graph', 'gdc_legacy_graph'.

Reply:
It matches the actual index structure, though... we have 4 different doc types stuffed into one graph index.

Reply:
as, @singergr mentioned, it's `<index>/<doc_type>.mapping.yaml` structure, it just happened that viz indices have only 1 `doc_type` 🤷‍♂